deployment/Dockerfile

```diff
@@ -6,6 +6,12 @@ RUN \
     bash \
     curl \
     jq \
-    zip
+    zip \
+    sudo
```

Comment:
We shouldn't need `sudo`, since we don't switch off the `root` user for those images.